src/Admin/AbstractAdmin.php

```diff
@@ -1747,8 +1747,6 @@ public function isCurrentChild(): bool
 
     /**
      * Returns the current child admin instance.
```

Comment:
Useless

Reply:
I will try to revert changes here, they are not needed for this PR. (will open more Pr for the rest of the directories)

Reply:
Reverted. We will get to Admin directory some day 😅